content-scripts/pages/components_page.tsx

```diff
@@ -0,0 +1,112 @@
+import jsx from "texsaur";
+import Button from "../components/Button";
+import { Table } from "../components/Table";
+import Icon from "../components/Icon";
+
+const components = ["Icon", "Button", "Table"];
+
+export function createComponentsPage() {
+  if (!document.location.href.toLowerCase().includes("components")) {
+    return;
+  }
```

Comment:
The reason to add the url here is exactly to provide the page in production, ahah
No one would see it, but if so, I think it can be interesting to show the work done in the project. Also, if people know what components should exist, they may know some to be replaced and told us.